packages/dev/docs/pages/react-spectrum/layout.mdx

```diff
@@ -0,0 +1,190 @@
+insert your content into via slots. This often happens automatically through the use of semantic elements,
+but can also be configured through props. For example, the [Dialog](Dialog.html) component accepts 
+[Heading](Heading.html), [Content](Content.html), and [Footer](Footer.html) elements as children and 
+will handle the layout automatically.
+
+## Flex
+
+The [flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox)
+layout model is a simple yet versatile method of laying out components in rows and columns.
+You can use it to build vertical or horizontal stacks, simple wrapping grids, and more. The [Flex](Flex.html)
+component can be used to create flexbox containers, and any React Spectrum component can be used as a child.
+In addition, flex layouts are automatically mirrored in right-to-left languages, and can be nested to create 
+more complex layouts.
+
+In addition to the properties widely supported by CSS, React Spectrum also shims the `gap` property, along
+with `rowGap` and `columnGap`. These properties make it much easier to build layouts
+with consistent space between each item. The gap can be defined with Spectrum [dimension variables](styling.html#dimension-values)
+to ensure consistency across applications, and allow the layout to adapt to different devices automatically.
+
+### Example
+
+This example shows a simple horizontal stack, with a gap between each item defined using a Spectrum dimension variable.
+
+```tsx example
+<Flex direction="column" width="size-2000" gap="size-100">
+  <View backgroundColor="celery-600" height="size-800" />
+  <View backgroundColor="blue-600" height="size-800" />
+  <View backgroundColor="magenta-600" height="size-800" />
+</Flex>
+```
+
+### Learn more
+
+You can learn more about `Flex` and see more examples on the [Flex](Flex.html) page. In addition,
+there are many great resources on the web for learning flexbox.
+
+* [The MDN guide to flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout) — full walkthrough
+of flexbox layout.
+* [A Complete Guide to Flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) — great reference for all of the
+properties supported by flexbox.
+
+## Grid
+
+[CSS grid](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Grid_Layout/Basic_Concepts_of_Grid_Layout) 
+is a powerful way to lay out elements in two dimensions. It can be used to to build full page application
+layouts, or smaller user interface elements. It is especially powerful because it allows you to build many types of 
+layouts without extra presentational elements, keeping your code clean and semantic. In addition, grid layouts
+are automatically mirrored in right-to-left languages.
+
+The [Grid](Grid.html) component can be used as a container to define a grid layout. Any React Spectrum component can be used as
```

Comment:
not sure what you mean. MenuTrigger passes through the button, which would become the grid child right?

Reply:
Yeah, I'm just saying that the props for the grid child need to be applied to the trigger. MenuTrigger doesn't pass props through to the trigger
not super important I don't think but might be confusing because we don't have a good way of indicating what components of ours result in dom and which don't
might be something to discuss later